jekyll/_cci2/scheduled-pipelines.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: classic-docs
+title: "Scheduled Pipelines"
+short-title: "Scheduled Pipelines"
+description: "Learn how to use scheduled pipelines"
+order: 20
+version:
+- Cloud
+- Server v3.x
+- Server v2.x
+suggested:
+  - title: Manual job approval and scheduled workflow runs
+    link: https://circleci.com/blog/manual-job-approval-and-scheduled-workflow-runs/
+  - title: How to trigger a workflow
+    link: https://support.circleci.com/hc/en-us/articles/360050351292?input_string=how+can+i+share+the+data+between+all+the+jobs+in+a+workflow
+  - title: Conditional workflows
+    link: https://support.circleci.com/hc/en-us/articles/360043638052-Conditional-steps-in-jobs-and-conditional-workflows
+---
+
+* TOC
+{:toc}
+
+## Overview
+{: #overview }
+
+Scheduled pipelines allow users to trigger pipelines periodically based on a schedule.
+
+Since the scheduled run is based on pipelines, scheduled pipelines have all the features that come with using pipelines:
+
+- Control the actor associated with the pipeline, which can enable the use of restricted contexts.
+- Use dynamic config via setup workflows.
+- Modify the schedule without having to edit `config.yml`.
+- Interact with auto-cancelling of pipelines.
+- Specify pipeline parameters associated with a schedule.
+- Consolidate the management of common schedules.
+
+Scheduled pipelines are configured through the API, or through the project settings in the CircleCI application.
+
+## Get started with scheduled pipelines in CircleCI
+{: #get-started }
+
+You have the option of setting up scheduled pipelines from scratch, or you can migrate existing scheduled workflows to scheduled pipelines.
+
+### Start from scratch
+{: #start-from-scratch }
+
+#### Use the API
+{: #api }
+{:.no_toc}
+
+If your project has no scheduled workflows and you would like to try out scheduled pipelines:
+
+1. Have your CCI token ready, or create a new token by following [these steps](https://circleci.com/docs/2.0/managing-api-tokens/).
+2. Create a new schedule using the API. For example:
+
+```sh
+curl --location --request POST 'https://circleci.com/api/v2/project/<project-slug>/schedule' \
+--header 'circle-token: <your-cci-token>' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "name": "my schedule name",
+    "description": "some description",
+    "attribution-actor": "system",
+    "parameters": {
+      "branch": "main"
+      <additional pipeline parameters can be added here>
+    },
+    "timetable": {
+        "per-hour": 3,
+        "hours-of-day": [1,15],
+        "days-of-week": ["MON", "WED"]
+    }
+}'
+```
+
+For additional information, refer to the **Schedule** section under the [open-api docs](https://circleci.com/docs/api/v2/).
+
+#### Use project settings
+{: #project-settings }
+{:.no_toc}
+
+1. In the CircleCI application, select the project, and navigate to **Project Settings**.
```

Comment:
People sometimes struggle to find project settings - I've tried to make it clear in the first step here, maybe use the same, or similar description? https://circleci.com/docs/2.0/managing-api-tokens/#creating-a-project-api-token